### PR TITLE
Bug 1069004 - Make brightness slider accessible.

### PR DESCRIPTION
**Pull request by eeejay**

### CODE REVIEW

**Comment by try-server-hook** [comment]
[eeejay Eitan Isaacson (eeejay)](https://github.com/eeejay) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=800edf0f7a13)

File: apps/gallery/js/ImageEditor.js [hunk 1]
```diff
@@ -223,6 +223,13 @@ var exposureSlider = (function() {
   thumb.addEventListener('touchend', function(e) {
     thumb.classList.remove('active');
   });
+  slider.addEventListener('keypress', function(e) {
+    if (e.keyCode == KeyEvent.DOM_VK_DOWN) {
+      setExposure(currentExposure - 0.25);
+    } else {
+      setExposure(currentExposure + 0.25);
+    }
+  });
```
Comment (punamdahiya) [inline apps/gallery/js/ImageEditor.js]: Is it possible to test the above code on flame-kk? I tried with screen reader enabled in settings that failed to get exposure slider to work.

Comment (eeejay) [inline apps/gallery/js/ImageEditor.js]: Yes, the key up/down is an unfortunate desktop convention. When you navigate to the slider swipe up and down with one finger to change the value.

Comment (punamdahiya) [inline apps/gallery/js/ImageEditor.js]: Please add comment above Line 226 to explain that it's screen reader specific and the purpose.

**Comment by try-server-hook** [comment]
[eeejay Eitan Isaacson (eeejay)](https://github.com/eeejay) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=96c94ed13a71)

**Pull request merged**